app/components/timer/utils.ts

```diff
@@ -0,0 +1,25 @@
+export function parseTimer(seconds: number): string {
+  if (!seconds) {
+    return "00:00"
+  }
+  const min = parseInt(String(seconds / 60)).toString()
+  const sec = parseInt(String(seconds % 60)).toString()
+  const minute = parseInt(min) < 10 ? `0${min}` : min
```

Comment:
Cleaner to use [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) here.